<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HousePrice</class>
 <widget class="QMainWindow" name="HousePrice">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2" colspan="3">
     <widget class="QTableWidget" name="MainTableForDocs">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>129</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>№ Отчёта</string>
       </property>
       <property name="background">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата регистрации</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата последнего изменения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Владелец</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Заказчик</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Адрес объекта</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Оценочная стоимость</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="search_layer">
      <property name="text">
       <string>Поиск...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="New_doc_Button">
      <property name="text">
       <string>Добавить отчёт</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QListWidget" name="sortirovshik_po_datam">
      <item>
       <property name="text">
        <string>Все</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>За месяц</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2025</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2024</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="copy_button">
      <property name="text">
       <string>Создать дубликат</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="delete_button">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1057</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="save_dir">
    <property name="title">
     <string>Путь сохранения</string>
    </property>
    <addaction name="action_dir"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="save_dir"/>
  </widget>
  <action name="action_dir">
   <property name="text">
    <string>Выбор папки</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>search_layer</tabstop>
  <tabstop>delete_button</tabstop>
  <tabstop>copy_button</tabstop>
  <tabstop>New_doc_Button</tabstop>
  <tabstop>MainTableForDocs</tabstop>
  <tabstop>sortirovshik_po_datam</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
